content/en/docs/tasks/debug/debug-cluster/audit.md

```diff
@@ -146,7 +146,7 @@ You can configure the log audit backend using the following `kube-apiserver` fla
 If your cluster's control plane runs the kube-apiserver as a Pod, remember to mount the `hostPath`
 to the location of the policy file and log file, so that audit records are persisted. For example:
 ```shell
-    --audit-policy-file=/etc/kubernetes/audit-policy.yaml \
+    --audit-policy-file=/etc/kubernetes/audit-policy.yaml
```

Comment:
In shell (which this is), the trailing backslash is required for continuation.

> A non-quoted backslash (\) is the escape character. It preserves the literal value of the next character that follows, with the exception of <newline>. If a \<newline> pair appears, and the backslash is not itself quoted, the \<newline> is treated as a line continuation (that is, it is removed from the input stream and effectively ignored).

(bash documentation)

In YAML, using a backslash can work like that too. For example:
```yaml
key: "command \
  --foo \
  --bar"
```
which is equivalent to:
```yaml
{
  "key": "command --foo --bar"
}
```

If you want JSON, you will need to adapt the line. That's legit.

Using shell wrapping communicates that it's all one command, and people can easily delete the backslash if they want to put the command on one line.

Reply:
Hey @sftim 

If talk about `shell` - yes, you're true. I'm talking about the YAML configuration file: 
```controlplane $ cat /etc/kubernetes/manifests/kube-apiserver.yaml | grep -i "command:" -A10
  - command:
    - kube-apiserver
    - --advertise-address=172.30.1.2
    - --allow-privileged=true
    - --authorization-mode=Node,RBAC
    - --audit-policy-file=/etc/kubernetes/audit-policy.yaml
    - --audit-log-path=/var/log/kubernetes/audit/audit.log
    - --client-ca-file=/etc/kubernetes/pki/ca.crt
    - --enable-admission-plugins=NodeRestriction
    - --enable-bootstrap-token-auth=true
    - --etcd-cafile=/etc/kubernetes/pki/etcd/ca.crt
controlplane $ ```

So, to add auditing there, I must use the off documentation https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/ where I should copy/past some parameters.

Reply:
Okay, This one is not Shell. It is YAML, from kube-apiserver.yaml.
line 148 is misleading.

Reply:
If this is an excerpt from YAML, it's **not** a valid YAML list (there should also be a `-` preceding each entry).

If making such a change we should also amend the text to make it clear that we are providing an excerpt from a manifest for a static Pod, as the current page doesn't imply or state that.

Reply:
right，the leading `-`s are missing.